src/components/FormSections/PersonalInfo.jsx

```diff
@@ -213,7 +214,7 @@ function PersonalInfo(props){
                             </div>
                             <div data-testid="formGroup" className="usa-form-group usa-form-group--day">
                                 <label data-testid="label" className="usa-label" htmlFor="date_of_birth_day">
-                                    Day
+                                <span dangerouslySetInnerHTML= {{__html: string.day}}/>
```

Comment:
we do not need the dangerouslysetinnerhtml
set to `{string.day}`

Reply:
removed the dangerouslysetinnerhtml and fixed format